siriuspy/siriuspy/devices/bpm.py

```diff
@@ -993,39 +998,86 @@ def mturn_config_acquisition(
             bpm.acq_nrsamples_pre = nr_points_before
             bpm.acq_nrsamples_post = nr_points_after
 
-        self.cmd_mturn_acq_start()
+        return self.cmd_mturn_acq_start()
 
-    def cmd_mturn_acq_abort(self) -> bool:
+    def cmd_mturn_acq_abort(self, wait=True, timeout=10) -> int:
         """Abort BPMs acquistion.
 
+        Args:
+            wait (bool, optional): whether or not to wait BPMs get ready.
+                Defaults to True.
+            timeout (int, optional): Time to wait. Defaults to 10.
+
         Returns:
-            bool: Whether or not abort was successful.
+            int: code describing what happened:
+                =0: BPMs are ready.
+                >0: Index of the first BPM which did not updated plus 1.
```

Comment:
updated -> update. This also appears in other docstrings

Reply:
Right. Done!